packages/server/src/integration-test/mysql.spec.ts

```diff
@@ -0,0 +1,1289 @@
+import fetch from "node-fetch"
```

Comment:
Would it be possible to avoid this much duplication? I would argue, its probably worthwhile just writing your test case for mysql into this for now, if at some point in the future we want to have a full test suite like the postgres test for all SQL datasources I think we'll need to write something generic which handles it, rather than a separate test case for each.

My preference for this would be to keep the postgres as the all singing all dancing test for SQL, then having the `mysql` test case here purely to test edge case stuff that differs specifically due to MySQL implementation (anywhere, we've needed to do an if mySQL do X).

Reply:
I can do - there may be a couple other tests worth keeping, for instance MySQL uses `DATABASE` instead of `SCHEMA` so there was a small implementation difference in a couple of the tests, but most of them were basically duplicates